src/sdk/http.ts

```diff
@@ -106,7 +106,7 @@ export class Http {
    * @param {string} url the URL path
    * @param {AxiosRequestConfig} config request configuration
    * @returns {Promise<AxiosResponse>} promise that resolves with server response
-   * @throws {ApiError} if the server responds with an error
+   * @throws {ApiError} if the server responds with an erroro some maintenance on the core right now?
```

Comment:
lets revert that, doesn't really make sense

Reply:
Wow, good catch. Don't know why my IDE did it. Doesn't make sense indeed.